tests/pipelines/test_pipelines_common.py

```diff
@@ -541,19 +541,18 @@ def test_load_default_pipelines_pt(self):
     @slow
     @require_tf
     def test_load_default_pipelines_tf(self):
-        import tensorflow as tf
-
+        from transformers.modeling_tf_utils import keras
         from transformers.pipelines import SUPPORTED_TASKS
 
-        set_seed_fn = lambda: tf.random.set_seed(0)  # noqa: E731
+        set_seed_fn = lambda: keras.utils.set_random_seed(0)  # noqa: E731
```

Comment:
So tf.random.set_seed doesn't actually set the seed for the TF environment? 

Does keras.utils.set_random_seed control just keras objects, or the whole TF env too?

Reply:
As far as I can tell, `keras.utils.set_random_seed()` basically sets a bunch of seeds, which includes `tf.random.set_seed()`, but also some others. It's the [currently recommended approach for reproducible initialization](https://keras.io/examples/keras_recipes/reproducibility_recipes/) in the Keras docs.

Reply:
Hope there is no other tests having the same init issue but no failure by luck 🤞 .
(otherwise we might get different results for them after this PR 😅 )

Reply:
I'm not surprised it's the recommended one in the keras docs ;) 

OK, well I just we just shrug and put it in the list of other TF quirks